shm/src/ship.cpp

```diff
@@ -19,11 +19,11 @@ Ship::Ship(int capacity,
       speed_(speed),
       name_(name),
       id_(id) {
-  TimeServiceLocator::getTime()->addObserver(this);
+    TimeServiceLocator::getTime()->addObserver(this);
 }
 
 Ship::Ship(int maxCrew, int speed, size_t id, Delegate* delegate)
-    : Ship(0, maxCrew, speed, "", id, delegate) {}
+    : Ship(100, maxCrew, speed, "", id, delegate) {}
```

Comment:
I think about removing this entire c-tor...

Reply:
We will remove that, however AFAIK it was a part of some homework oop1 - so i will leave it until official PR.